capstone-finder.md

```diff
@@ -0,0 +1,61 @@
+# Capstone Finder
+
+> [!WARNING]
+> This is a work in progress. Do not merge this to the main branch.
+
+## Overview
+
+This project aims to help students find members for their capstone projects. Students can search for members by branch, gender and GPA, three of the criteria that are required for diversity in capstone groups.
+
+---
+
+## Features
+
+- Search for members by branch, gender and GPA
+- Add resume to your profile
+- View member profiles
+- Chat with members before giving out contact information
+- Auto delete chats after 1 week
+- Auto delete account after 6 months of inactivity
+
+---
+
+## Basic Tech Stack
+
+- Next.js
+- Tailwind CSS
+- Shadcn UI
+- MongoDB as database
+- `jose` for JWT
+- `pesu-auth` for authentication
+
+---
+
+## Workflows
+
+### Student Signup
+
+- Student uses PESUAcademy credentials to sign in to `pesu-auth`
```

Comment:
ideally, by the time you start developing this, i'll have the pesu-oauth2 up. so use that instead